server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -212,15 +217,15 @@ public void createPendingSegmentsTable(final String tableName)
                 + "  dataSource VARCHAR(255) NOT NULL,\n"
                 + "  created_date VARCHAR(255) NOT NULL,\n"
                 + "  start VARCHAR(255) NOT NULL,\n"
-                + "  \"end\" VARCHAR(255) NOT NULL,\n"
+                + "  %send%s VARCHAR(255) NOT NULL,\n"
                 + "  sequence_name VARCHAR(255) NOT NULL,\n"
                 + "  sequence_prev_id VARCHAR(255) NOT NULL,\n"
                 + "  sequence_name_prev_id_sha1 VARCHAR(255) NOT NULL,\n"
                 + "  payload %2$s NOT NULL,\n"
```

Comment:
The `%2$s` is positional and needs to be adjusted since payload type moved from position 2 to position 4. Or, alternatively keep this the same, but move getQuoteString to the end and use `%3$send%3$s` above rather than `%send%s`.